### DISCUSSION TITLE
Edit file feature?

### DISCUSSION THREAD

**Discussion by farisc0de** [community/community#13841] [Mobile] [answered]
Hi

I was wondering if you could add edit a file feature

Like if I can edit a code on the go or update a read me file when needed

I hope you consider adding it to the mobile app

Thank you

**Comment by eYeMirR0rr0RriMeYe** [comment]
@github-staff great idea!

**Comment by brianlovin** [comment]
Thanks for the note! This is definitely one of the most requested features, so I'm going to add your +1 to our tracking issue 😊

**Comment by spaltrowitz** [comment, marked as answer]
thanks for your suggestion @farisc0de! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code
